### ISSUE TITLE
Testing for zero-sized fonts on elements containing images

### ISSUE THREAD

**Issue by fstorr** [dequelabs/axe-core#594]
I discovered during an accessibility audit that an element with `font-size:0` set on it (for example, a `<figure>`) won't display `alt` text if an image fails to load. This happens in Chrome, Firefox, and Safari. Edge 14 (I can't test on a build higher than that) doesn't have this problem.

I created a [test page on CodePen](https://codepen.io/fstorr/full/LzKgGj/) to demonstrate this.

**Comment by marcysutton** [comment]
Thanks for filing @fstorr! To clarify, did you find that it also prevented AT from reading out the alt text? That ups the priority of fixing this in axe-core, more than a general image load fallback issue.

**Comment by fstorr** [comment]
I found that JAWS, NVDA, and VoiceOver all still announced the alt text when the image was broken, so users of screen readers won't be affected by this.

**Comment by marcysutton** [comment]
Hmmm. I don't know that it's necessarily an accessibility problem then, per se. It's more of a usability/progressive enhancement problem. I make that distinction since we have to prioritize what we add and maintain in axe-core. @WilcoFiers, any input on this?

**Comment by WilcoFiers** [comment]
I agree. Doesn't seem like an accessibility issue. I think we can close this.

**Comment by fstorr** [comment]
Thanks for the feedback :) I'm going to file bugs with browser vendors as I see this as an accessibility issue affecting users who can't, or have made the decision not to, load images.
